src/routers/user.js

```diff
@@ -79,8 +79,19 @@ router.get("/signup", async (req, res) => {
 router.get("/users", async (req, res) => {
     try {
         const city = req.query.city
-        const restaurants = await User.find({city})
-        res.render('searchByCity', { restaurants })
+        const name = req.query.name
+
+        if (!city){
+            var restaurants = await User.find({ name })
```

Comment:
What if name is also empty?

Reply:
It wont show anything anyways

Reply:
Yes but it should be explicity handled. These cases should be handled properly